types/tapplet/signer.ts

```
@@ -0,0 +1,35 @@
+}
+
+export type WindowSize = {
+  width: number
+  height: number
+}
+
+export type PickMatching<T, V> = {
+  [K in keyof T as T[K] extends V ? K : never]: T[K]
+}
+/* eslint-disable @typescript-eslint/no-unsafe-function-type */
+export type ExtractMethods<T> = PickMatching<T, Function>
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)